src/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.cs

```diff
@@ -121,11 +390,119 @@ public partial class NullableConverter : System.ComponentModel.TypeConverter
+        public abstract void SetValue(object component, object value);
+        public abstract bool ShouldSerializeValue(object component);
+    }
+    public partial class PropertyDescriptorCollection : System.Collections.ICollection, System.Collections.IEnumerable, System.Collections.IDictionary, System.Collections.IList
+    {
+        public static readonly System.ComponentModel.PropertyDescriptorCollection Empty;
+        public PropertyDescriptorCollection(System.ComponentModel.PropertyDescriptor[] properties) { }
+        public PropertyDescriptorCollection(System.ComponentModel.PropertyDescriptor[] properties, bool readOnly) { }
+        public int Count { get; }
+        bool System.Collections.ICollection.IsSynchronized { get; }
+        object System.Collections.ICollection.SyncRoot { get; }
+        bool System.Collections.IDictionary.IsFixedSize { get; }
+        void System.Collections.IDictionary.Add(object key, object value) { }
+        bool System.Collections.IDictionary.Contains(object key) { return default(bool); }
+        object System.Collections.IDictionary.this[object key] { get { return default(object); } set { } }
+        void System.Collections.IDictionary.Remove(object key) { }
+        bool System.Collections.IDictionary.IsReadOnly { get; }
+        System.Collections.IDictionaryEnumerator System.Collections.IDictionary.GetEnumerator() { return default(System.Collections.IDictionaryEnumerator); }
+        System.Collections.ICollection System.Collections.IDictionary.Keys { get; }
+        System.Collections.ICollection System.Collections.IDictionary.Values { get; }
+        bool System.Collections.IList.IsFixedSize { get; }
+        bool System.Collections.IList.IsReadOnly { get; }
+        int System.Collections.IList.Add(object value) { return default(int); }
+        void System.Collections.IList.Remove(object value) { }
+        bool System.Collections.IList.Contains(object value) { return default(bool); }
+        int System.Collections.IList.IndexOf(object value) { return default(int); }
+        void System.Collections.IList.Insert(int index, object value) { }
+        object System.Collections.IList.this[int index] { get { return default(object); } set { } }
+        public virtual System.ComponentModel.PropertyDescriptor this[int index] { get { return default(System.ComponentModel.PropertyDescriptor); } }
+        public virtual System.ComponentModel.PropertyDescriptor this[string name] { get { return default(System.ComponentModel.PropertyDescriptor); } }
+        public int Add(System.ComponentModel.PropertyDescriptor value) { return default(int); }
+        public void Clear() { }
+        public bool Contains(System.ComponentModel.PropertyDescriptor value) { return default(bool); }
+        public void CopyTo(System.Array array, int index) { }
```

Comment:
We discussed this at the design review based on your earlier comments in the issue, but it was determined that this contract is mostly for ease of porting and thus we'll keep it as is. See https://github.com/dotnet/apireviews/pull/32 for more details